java/src/main/java/Main.java

```diff
@@ -0,0 +1,9 @@
+
```

Comment:
It looks like there might be a few extra blank lines floating around these files, it would be good if we could keep the style consistent without them

Reply:
Ok, done